### PR TITLE
E4defrag fix

### PR DESCRIPTION
**Pull request by mrqwer88**
Build in rpm correctly and looking working for me.

- backporting commit with [OVZ-6716](https://bugs.openvz.org/browse/OVZ-6716) fix.
- Add --defrag flag to usage
- Bump version

### CODE REVIEW

**Comment by kolyshkin** [comment]
1. Please remove the last commit, I will release a new version myself.
2. Please also add --defrag flag to the ploop.8 man page.

**Comment by mrqwer88** [comment]
Fixed - last commit with bump version removed.
In previous commit added flag to man page.

**Comment by mrqwer88** [comment]
Any news?

As variant may be more good create pool from another branch - https://github.com/mrqwer88/ploop/commits/backport with more changes?

File: tools/ploop.8.in [hunk 1]
```diff
@@ -776,6 +778,11 @@ If not, one can use
 .B --automount
 option to automatically mount ploop for the duration of the operation.
 
+Option 
+.B --defrag
+can be used to run ext4 defragmetation via /usr/sbin/e4defrag2 utility,
```
Comment (kolyshkin) [inline tools/ploop.8.in]: A typo: defragmetation -> defragmentation

File: tools/ploop.8.in [hunk 2]
```diff
@@ -776,6 +778,11 @@ If not, one can use
...
+befor discard unused blocks from a ploop image.
```
Comment (kolyshkin) [inline tools/ploop.8.in]: A typo: befor -> before

**Comment by kolyshkin** [comment]
Please fix typos in man page, otherwise LGTM

**Comment by kolyshkin** [comment]
Also, if the patches are not yours, don't put yourself as the author

**Comment by kolyshkin** [comment]
OK, I have fixed your patch as per my comments above and committed it:
  https://github.com/kolyshkin/ploop/commit/675ca48eeefe5c536d2bae4506819eb1b20bcce7
The other patch you have here (from Igor) I backported as is.

So this pull req can now be closed. Thanks for your contribution!

**Comment by kolyshkin** [comment]
Speaking of https://github.com/mrqwer88/ploop/commits/backport,
I have backported all the commits you have there at the moment,
except for ploop-copy one which is irrelevant. The thing is, there
are two different implementations of ploop-copy, the one that is
used in OpenVZ is in lib/pcopy.c, the one used in Virtuozzo is in
lib/ploop-copy.c. 'ploop copy' command line tool (used by OpenVZ's
vzmigrate) is using the pcopy.c implementation, i.e. the other one is
unused, so I chose to not backport patches to it.

**Comment by mrqwer88** [comment]
Thanks a lot!

A little question about - 
https://github.com/mrqwer88/ploop/commit/5f81138172fce95e6b4a013882c2b327b103778a
After "ploop_mount_fs: don't mount ro when finding balloon inode" https://github.com/kolyshkin/ploop/commit/0606858ec205fb3652e1578a1449d2341fa72759  changeв the behavior when you mount it via ploop/vzctl.
And it looks not good for openvz-legacy ploop, because it broken some users script and custom tools.
May be it was more good to create balloon file, if it not exist(after some operations or on old containers after convertation from simfs with old versions software).

**Comment by mrqwer88** [comment]
Sorry - this version already have another ploop_mount_fs without 
```
	if (stat(buf, &st) < 0) {
		ploop_err(errno, "Can't stat balloon file %s", buf);
		umount(param->target);
		return SYSEXIT_MOUNT;
	}
```

Thanks!

**Pull request closed without merging**